json-endpoints/jetpack/class.jetpack-json-api-cron-endpoint.php

```diff
@@ -0,0 +1,118 @@
+<?php
+
+
+class Jetpack_JSON_API_Cron_Endpoint extends Jetpack_JSON_API_Endpoint {
+	protected $needed_capabilities = 'manage_options';
+
+	protected function validate_call( $_blog_id, $capability, $check_manage_active = true ) {
+		parent::validate_call( $_blog_id, $capability, false );
+	}
+	protected function result() {
+
+	}
+
+}
+// GET /sites/%s/cron
+class Jetpack_JSON_API_Cron_Get_Endpoint extends Jetpack_JSON_API_Cron_Endpoint {
+	protected $needed_capabilities = 'manage_options';
+
+	protected function validate_call( $_blog_id, $capability, $check_manage_active = true ) {
+		parent::validate_call( $_blog_id, $capability, false );
+	}
+
+	protected function result() {
+		$cron = _get_cron_array();
```

Comment:
No need to have this defined as a variable if we're only returning it in the response:

```php
		return array(
			'cron_array' => _get_cron_array(),
			'current_timestamp' => time()
		);
```